parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -518,6 +503,8 @@ finally
     Write-Log "Generate CSE result to $CSEResultFilePath : $global:ExitCode"
     echo $global:ExitCode | Out-File -FilePath $CSEResultFilePath -Encoding utf8
 
+    Logs-To-Event -TaskName "AKS.WindowsCSE.cse_main" -TaskMessage "ExitCode: $global:ExitCode. ErrorMessage: $global:ErrorMessage." 

```

Comment:
Could you help to update `Postpone-RestartComputer` to log the restart event if it is called?